runtime/bcverify/staticverify.c

```diff
@@ -1794,6 +1799,12 @@ j9bcv_verifyClassStructure (J9PortLibrary * portLib, J9CfrClassFile * classfile,
 				/* TODO: determine if this should be verifyError */
 				goto _formatError;
 			}
+#ifdef J9VM_OPT_VALHALLA_VALUE_TYPES
```

Comment:
Check for `CFR_METHOD_NAME_NEW` can be added into line 1774 if `J9VM_OPT_VALHALLA_NEW_FACTORY_METHOD` is defined.